Reports/Conceptual-Design.md

```diff
@@ -0,0 +1,331 @@
+
+#### Budget
+
+|Item|Cost per Item|Quantity|Total Cost for Item|
+| :- | :- | :- | :- |
+|Brushed Motors (24V)|~$60.00|2|$120.00|
+|Motor Microcontroller|~$30.00|1|$30.00|
+|Motor Drivers (L298N H-bridge module)|$10.00|2|$20.00|
+|Optical Encoders|$20.00|2|$40.00|
+|Total|||$210.00|
+
+#### Skills
+
+For the Motor Control subsystem, there is mainly hardware implementation, but software is required for PID control. A team member with theoretical knowledge of PID feedback loops and technical experience in programming would be well-suited for this role. Classes in Controls and Robotics would help serve as a sufficient background for the project applications. 
+
+### Power Management
+#### Atomic Subsystem Specifications
+
+There are several quantifiable specifications and constraints for the power subsystem given the competition rules and the adherence to safe engineering practice. Procedurally, the main areas of focus are the start and stop mechanisms during play. In accordance with Specification 2-ii and 6, the robot functionality may have a trigger based on a given Start LED controlled by the referee or through a clearly labeled electrical start button. For the manual start button, the robot shall not move for five seconds after activation, but onboard commands can run during this time in preparation for play. The robot shall be able to safely and efficiently pause all its functions using a clearly labeled emergency stop button according to Specification 7. If there are multiple independently acting ‘Robot Units’, each shall have their own emergency stop button. Regarding power system requirements, the robot shall not go beyond 30V, as given by Specification 12. It shall also implement proper circuit shielding and isolation based on the disturbances from background interference in the competition environment, as given by Specification 11. Constraint 3 deals with International Electrotechnical Commision regulations for power supply, emergency stop, and circuit protection requirements. From this standard, the circuitry of the robot shall account for and add appropriate devices to promote safety measures such as overcurrent, overload, and over-temperature protection [3]. The power subsystem of this robot will be the electrical backbone of the project. Without a structured model of power management and distribution, the robot would not be able to meet all functions, specifications and constraints. For this subsystem to work correctly with the scope of the project, we must install a battery/(ies) so that the robot can be powered without a cord attached to a wall socket.
```

Comment:
List "SHALL" statements.

Reply:
Format changed to listed SHALL statements using the given specifications and constraints.